build.sbt

```diff
@@ -231,11 +236,9 @@ lazy val publishSettings = Seq(
       "scm:git:git@github.com:laserdisc-io/laserdisc.git"
     )
   ),
-  homepage := Some(url("http://laserdisc.io")),
   licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/laserdisc-io/laserdisc/master/LICENSE")),
   pgpPublicRing := file(".travis/local.pubring.asc"),
-  pgpSecretRing := file(".travis/local.secring.asc"),
-  releaseEarlyWith := SonatypePublisher
+  pgpSecretRing := file(".travis/local.secring.asc")
```

Comment:
This line we shouldn’t need anymore, same for the file `.travis/secrets.tar.enc`